### ISSUE TITLE
Required version of 'minify' does not support commonjs import

### ISSUE THREAD

**Issue by MalikKillian** [coderaiser/node-mollify#7]
Mollify 4.0.0 is a CommonJS module but minify ^8.0.3 is an ES module. Running a lightly modified version of the example code I received an error.

```bash
$ cat test.js
const mollify = require('./lib/mollify');
const http = require('http');
const express = require('express');

const app = express();
const server = http.createServer(app);

const port = 1337;
const ip = '0.0.0.0';

app.use(mollify({
    dir: __dirname,
    is: true, // default
}));

app.use(express.static(__dirname));

server.listen(port, ip);
$ node test.js
/home/user/repo/github/coderaiser/node-mollify/lib/mollify.js:4
const minify = require('minify');
               ^

Error [ERR_REQUIRE_ESM]: require() of ES Module /home/user/repo/github/coderaiser/node-mollify/node_modules/minify/lib/minify.js from /home/user/repo/github/coderaiser/node-mollify/lib/mollify.js not supported.
Instead change the require of minify.js in /home/user/repo/github/coderaiser/node-mollify/lib/mollify.js to a dynamic import() which is available in all CommonJS modules.
    at Object.<anonymous> (/home/user/repo/github/coderaiser/node-mollify/lib/mollify.js:4:16)
    at Object.<anonymous> (/home/user/repo/github/coderaiser/node-mollify/test.js:1:17) {
  code: 'ERR_REQUIRE_ESM'
}
```
Seems like it's time for mollify to become an ES module. If I modify the code to use `import` and fix the export statement things work fine.
```bash
$ cat test.js
import mollify from './lib/mollify.js';
import http from 'http';
import express from 'express';

import {fileURLToPath} from 'url';
import {dirname} from 'path';

const __filename = fileURLToPath(import.meta.url);
const __dirname = dirname(__filename);

const app = express();
const server = http.createServer(app);

const port = 1337;
const ip = '0.0.0.0';

app.use(mollify({
    dir: __dirname,
    
    // default
    is: true,
}));

app.use(express.static(__dirname));

server.listen(port, ip);
$ cat ./lib/mollify.js | head -n10
import path from 'path';
import {minify} from 'minify';
import ponse from 'ponse';

export default (options) => {
    if (!options)
        options = {};
    
    return mollify.bind(null, options);
};
```
And the "type" in package.json needs to be "module" instead of "commonjs".

**Comment by coderaiser** [comment]
Are you willing for a PR?

**Comment by MalikKillian** [comment]
Let me fork and see what I can do.

**Comment by MalikKillian** [comment]
https://github.com/coderaiser/node-mollify/pull/8

**Comment by coderaiser** [comment]
Landed in [v5.0.0](https://github.com/coderaiser/node-mollify/releases/tag/v5.0.0) 🎉 , thanks a lot!

Is it works for you?

**Comment by MalikKillian** [comment]
We good. Thanks for the quick responses!
